index.html

```diff
@@ -207,15 +207,15 @@ <h2>Terminology</h2>
 <p>The term <dfn>resource</dfn> is used to refer to elements and
 any other user-initiated fetches throughout this specification. For
 example, a resource could originate from XMLHttpRequest objects
-[[XHR]], HTML elements [[HTML5]] such as iframe, img,
-script, object, embed, and link with the link type of stylesheet,
-and SVG elements [[SVG11]] such as svg.</p>
+[[XHR]], HTML elements [[HTML5]] such as iframe, img, script,
```

Comment:
nit: s/HTML/HTML5

Reply:
You mean s/[[HTML5]]/[[HTML]]? I've made this change in the latest commit. Thanks!

Reply:
Awesome, thanks!